.travis.yml

```diff
@@ -1,8 +1,9 @@
 sudo: false
 language: node_js
 node_js:
-  - "4"
   - node
+  - 6
+  - 4
```

Comment:
@d3viant0ne Thx I will update asap 🚀 , webpack v1 is unnecessary, since v2 introduces breaking changes incompatible with v1 (to my knowledge 😛 )